frontend/src/components/Navigation/Navigation.js

```diff
@@ -1,10 +1,38 @@
 import React from 'react';
-import { Container, Nav } from './Navigation.styled';
+import { FaHome, FaBell, FaUserAlt, FaHeart, FaPlusSquare } from 'react-icons/fa';
+
+import {
+  Container,
+  NavMenu,
+  NavItem,
+  NavLink,
+  NavIcon,
+  NavButton,
+  NavText,
```

Comment:
Trochę mi się to `Nav` dubluje w oczach. Zastanawiam się, czy jest to potrzebne wszędzie jako przedrostek, skoro i tak każdy komponent dotyczy `Nav`.

Reply:
Słusznie!